json-path/src/main/java/com/jayway/jsonpath/internal/JsonContext.java

```diff
@@ -146,8 +146,8 @@ public String jsonString() {
         if(jsonPath != null){
         	return read(jsonPath);
         } else {
-                jsonPath = compile(path, filters);
-                cache.put(cacheKey, jsonPath);
+		jsonPath = compile(path, filters);
```

Comment:
I think the original formatting used spaces. See https://github.com/jayway/JsonPath/pull/335/files#diff-ebfcf97b9fab6bc1ac4c9e6c82e4b9dbR149